README.md

```diff
@@ -26,7 +26,7 @@ The OpenTrials API service.
 2. Run `npm install`;
 3. Run `npm run migrate`;
 4. (Optional) If you want, you can add some seed data using `npm run seed`;
-5. Run `npm run reindex`;
+5. (If you ran 4) Run `npm run reindex`;
```

Comment:
You're right: it won't. Created an issue for the reason our `npm run reindex` fails with an empty DB. It's a bug on BookshelfJS: https://github.com/tgriesser/bookshelf/issues/1461